### DISCUSSION TITLE
When I run the roslaunch command, an RLException error occurs!

### DISCUSSION THREAD

**Discussion by Jun-gil** [community/community#45397] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The text is the same as the title.
I tried various methods on google to solve this error. But it doesn't work, so I need your help.
Anyone is welcome, please help me to solve this error.
-----------------------------------<command>---------------------------------
awesometech@awesometech:~/catkin_ws$ roslaunch urs_wearable world.launch
-------------------------------------<result>-----------------------------------
... logging to /home/awesometech/.ros/log/165b93fc-9d30-11ed-967c-a99cf3fa13f4/roslaunch-awesometech-4851.log
Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

RLException: while processing /home/awesometech/catkin_ws/src/URSSimEnvironments/urs_wearable/launch/world.launch:
while processing /home/awesometech/catkin_ws/src/URSSimEnvironments/urs_wearable/launch/world.launch:
..
Invalid tag: maximum recursion depth exceeded while calling a Python object.

Arg xml is
The traceback for the exception was written to the log file

--------------------------------------------------------------------------------

this is my code I think there is no error in this code.

`<?xml version="1.0"?>`
`<launch>`
`<include file="$(find urs_wearable)/launch/world.launch">`
`<arg name="world" value="$(find urs_wearable)/worlds/test.world" />`
`<arg name="reference_latitude" value="32.319939" />`
`<arg name="reference_longitude" value="-106.763657" />`
`</include>`
`</launch>`

please help me......

**Comment by jge162** [comment]
good day @Gwakjoongil Based on the code you've provided, it appears that the error is caused by an infinite loop in the <include> tag in your launch file. This is caused by the fact that you are including the same launch file that you are currently running.

You can resolve this issue by changing the include file to another launch file or package that does not cause this recursion.

**Reply by Jun-gil** [reply]
Does the infinite loop exist in the code above? I couldn't confirm. Can you explain in a bit more detail?

**Reply by jge162** [reply]
yes

**Reply by jge162** [reply]
you are including the same launch file that you are currently running, which causes an infinite loop. Each time the launch file is included, it includes itself again, creating an infinite recursion.

**Reply by Jun-gil** [reply]
I do not have the same startup files. There is only one world.launch file.....what should i do now?

**Reply by jge162** [reply]
Please check the path of the include file in your launch file and make sure it is pointing to the correct location.

Check the log file that is mentioned in the error message to see if it provides more information about the error. 
The log file is located at /home/awesometech/.ros/log/165b93fc-9d30-11ed-967c-a99cf3fa13f4/roslaunch-awesometech-4851.log

**Reply by Jun-gil** [reply]
All of the code in the executable is pointing to the correct location. I also confirmed that the log file contains details about the error. But I couldn't find a solution to the error in the log file. I'm going crazy about what the hell is wrong.

**Reply by jge162** [reply]
yeah idk, so many variables. It could be that there is an issue with the package or dependencies that the launch file is trying to use. One way to check this would be to run the package and its dependencies independently to see if they are working as expected. try running a different world file. I really dont know the issue.

**Reply by Jun-gil** [reply]
Let's try. If the problem is solved, I will leave an answer in the comments. Thank you so much for kindly answering my question.

**Reply by jge162** [reply]
I think you got it. good luck

**Comment by jge162** [comment]
this ticket##45392 was solved. Please close repeated request ticket#45397 thank you.  not github related...

**Reply by Jun-gil** [reply]
#45392 is a different error than this one. Posted separately because #45392 didn't fix the error left in the comment.

**Reply by jge162** [reply]
ok
